src/browser/ui/dom/CSSPropertyOperations.js

```diff
@@ -64,6 +68,33 @@ if (__DEV__) {
       name.charAt(0).toUpperCase() + name.slice(1) + '?'
     );
   };
+
+  var warnStyleValueWithSemicolon = function(name, value) {
+    if (warnedStyleValues.hasOwnProperty(value) && warnedStyleValues[value]) {
+      return;
+    }
+    
+    warnedStyleValues[value] = true;
+    warning(
+      false,
+      'Style property values shouldn\'t contain a semicolon. Try "' +
+      name + ': ' + value.replace(badStyleValueWithSemicolonPattern, '') + '" instead.'
```

Comment:
I know you're just copying the pattern from above, but let's make use of `%s` instead of doing the concats (we're going to fix the others up in #2941)